staff/manuel-galvez/blog-layout/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Blog Layout</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Lato&display=swap" rel="stylesheet"> 
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="logo">
+                <img src="https://www.skylabcoders.com/images/403/default.png">
+            </div>
+            <ul>
+                <li><a href="#">Courses</a></li>
+                <li><a href="#">Community</a></li>
+                <li><a href="#">Events</a></li>
+                <li><a href="#">About</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <aside>
+            <div class="profile-container">
+                <div class="profile-img">
+                    <img src="https://is5-ssl.mzstatic.com/image/thumb/Video118/v4/3a/da/d9/3adad999-6ed0-de80-6b7e-8fccca15f42e/source/560x315mv.jpg">
+                </div>
+                <div class="about-me-header">About me</div>
+                <div class="about-me-content">
+                        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500...
+                </div>
+            </div>
+        </aside>
+        <div class="article-container">
+            <article>
+                <section class="article-img">
```

Comment:
You can use a `<figure>` element instead of `<section>` for images

Reply:
Didn't know. Changed it already. Thanks!